app/lib/application_extension.rb

```diff
@@ -23,6 +23,12 @@ def confirmation_redirect_uri
     redirect_uri.lines.first.strip
   end
```

Comment:
@ClearlyClaire I'm not sure what this property is, but it looks odd. Should probably be `redirect_uri.split.first`